lms/djangoapps/teams/static/teams/js/views/topic_teams.js

```diff
@@ -32,9 +33,10 @@
                 render: function() {
                     var self = this;
                     this.collection.refresh().done(function() {
+                        var messageHtml;
                         TeamsView.prototype.render.call(self);
                         if (self.canUserCreateTeam()) {
-                            var message = interpolate_text(
+                            messageHtml = HtmlUtils.interpolateHtml(
```

Comment:
Ok.  It's too bad i18n makes things so complicated.  It would be nice if all the html could be in the templates.